config/initializers/assets.rb

```diff
@@ -17,4 +17,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
```

Comment:
Don't think this change is needed. admin.js and admin.css are needed for ActiveAdmin stuff, the new JS file should already be in the asset pipeline due to its location